### DISCUSSION TITLE
Light theme by default

### DISCUSSION THREAD

**Discussion by kenara** [mastodon/mastodon#21888] [General]
I'd like my server to use light theme by default, and let users switch to dark if they like.

How do I do that?

I tried changing themes.yml like so:

```
default: styles/mastodon-light.scss
#default: styles/application.scss
contrast: styles/contrast.scss
mastodon-light: styles/mastodon-light.scss
```
and restarting but that didn't help.

.... (later) figured it out

**Comment by mjankowski** [comment]
Looks like you self-sorted this out.

If something could be made more clear in docs, can you open an issue in that repo?
